App/src/main/java/com/google/sps/servlets/ShelterMatch.java

```diff
@@ -49,41 +53,116 @@ public void addShelters(){
         nameFive.setPhoneNum(2137448186L);
         nameFive.setDescription("9-18 month program for veterans with mental health issues");
         nameFive.setAgeRange(18,100);
+        nameFour.setDisability(false);
+        list.add(nameFive);
+
+        Shelter nameSix = new Shelter();
+        nameSix.setName("Downtown Women’s Center: Permanent Supportive Housing");
+        nameSix.setLocation("442 S. San Pedro Street, Los Angeles, CA 90013");
+        nameSix.setPhoneNum(2136800600L);
+        nameSix.setDescription("Permanent program for homeless women.");
+        nameSix.setAgeRange(18,100);
+        nameSix.setGender("female");
+        nameFour.setDisability(false);
+        list.add(nameSix);
+
+        Shelter nameSeven = new Shelter();
+        nameSeven.setName("Downtown Women’s Center: Community-Based Housing Program");
+        nameSeven.setLocation("442 S. San Pedro Street, Los Angeles, CA 90013");
+        nameSeven.setPhoneNum(2136800600L);
+        nameSeven.setDescription("Nightly program for homeless women and children.");
+        nameSeven.setAgeRange(18,100);
+        nameSeven.setGender("female");
+        nameFour.setDisability(false);
+        list.add(nameSeven);
+
+        Shelter nameEight = new Shelter();
+        nameEight.setName("Higher Goals");
+        nameEight.setLocation("10510 S Vermont Ave, Los Angeles, CA - 90044");
+        nameEight.setPhoneNum(3237559702L);
+        nameEight.setDescription("6 month program to get you back on your feet.");
+        nameEight.setAgeRange(18,100);
+        nameEight.setDisability(false);
         list.add(nameFive);
     }
 
-    public static void sortShelters(Shelter[] shelters){
-        for(int i = 0; i < shelters.length; i++){
+    //returns a Shelter[] of the sorted results based on score
+    public static Shelter[] sortTop3(){
+        Shelter[] output = new Shelter[3];
+        ArrayList<Shelter> temp = new ArrayList<Shelter>();
 
+        //populating temp with Shelters from list so temp can be altered without effecting list
+        for(int i = 0; i < list.size(); i++){
+            temp.add(list.get(i));
         }
+        
+        int count = 0;
+        while(count <= output.length-1){
+            int posMax = 0;
+            int maxScore = temp.get(0).getScore();
 
+            for(int j = 0; j < temp.size(); j++){
```

Comment:
This code works fine, and for the input size that we have it would be a waste of time to try to optimize it. That said...

If you want to get some practice honing your algorithms skills: What's the runtime of this algorithm? Can we improve it? (Feel free to ignore, just adding it here as an optional challenge.)